bioimageio_core_version: 0.5.3.post2
bioimageio_spec_version: 0.4.5.post7
rdf_sha256: 131fd061bf22238fdc122c9ea1cfa7f65d035c8040a27026a75ebc2765258352
status: failed
tests:
  bioimageio:
  - bioimageio_spec_version: 0.4.5post7
    error: null
    name: bioimageio.spec static validation of model RDF 0.4.5 with update to latest
      format version
    nested_errors: {}
    status: passed
    traceback: null
    warnings: {tags: 'Missing tags for categories: [{''software'': [''ilastik'', ''imagej'',
        ''fiji'', ''imjoy'', ''deepimagej'', ''n2v'']}]'}
  - bioimageio_spec_version: 0.4.5post7
    error: null
    name: bioimageio.spec static validation of model RDF 0.4.5
    nested_errors: {}
    status: passed
    traceback: null
    warnings: {tags: 'Missing tags for categories: [{''software'': [''ilastik'', ''imagej'',
        ''fiji'', ''imjoy'', ''deepimagej'', ''n2v'']}]'}
  - bioimageio_core_version: 0.5.3post2
    bioimageio_spec_version: 0.4.5post7
    error: cannot reshape array of size 2924253 into shape (1,304,512,33)
    name: reproduced test outputs from test inputs
    status: failed
    traceback: ["  File \"/usr/share/miniconda/envs/6338615/lib/python3.7/site-packages/bioimageio/core/resource_tests.py\",
        line 127, in test_resource\n    expected = [np.load(str(out_path)) for out_path
        in model.test_outputs]\n", "  File \"/usr/share/miniconda/envs/6338615/lib/python3.7/site-packages/bioimageio/core/resource_tests.py\",
        line 127, in <listcomp>\n    expected = [np.load(str(out_path)) for out_path
        in model.test_outputs]\n", "  File \"/usr/share/miniconda/envs/6338615/lib/python3.7/site-packages/numpy/lib/npyio.py\",
        line 453, in load\n    pickle_kwargs=pickle_kwargs)\n", "  File \"/usr/share/miniconda/envs/6338615/lib/python3.7/site-packages/numpy/lib/format.py\",
        line 785, in read_array\n    array.shape = shape\n"]
  deepimagej:
  - bioimageio_core_version: 0.5.3post2
    bioimageio_spec_version: 0.4.5post7
    error: 'unsupported operand type(s) for +: ''PosixPath'' and ''str'''
    name: reproduce test outputs with deepimagej 2.1.13 using tensorflow_saved_model_bundle
      weights
    reason: error
    status: failed
    traceback: ["  File \"/home/runner/work/models/models/scripts/check_rdf_and_test_model.py\",
        line 122, in test_resource\n    error = (error or \"\") + download_deepimagej_model(fiji_path,
        rdf_path)\n", "  File \"/home/runner/work/models/models/scripts/utils.py\",
        line 27, in download_deepimagej_model\n    os.makedirs(fiji_dir + \"//models\")\n"]
  ilastik:
  - {bioimageio_core_version: 0.5.1, bioimageio_spec_version: 0.4.5post3, error: null,
    name: reproduce test outputs with ilastik <todo version> (draft) using tensorflow_saved_model_bundle
      weights, reason: null, status: passed, traceback: null}
  - {bioimageio_core_version: 0.5.3post2, bioimageio_spec_version: 0.4.5post6, error: null,
    name: reproduce test outputs with ilastik <todo version> (draft) using tensorflow_saved_model_bundle
      weights, reason: null, status: passed, traceback: null}
